xml/System.IO/WindowsRuntimeStorageExtensions.xml

```diff
@@ -189,11 +359,83 @@
         <Parameter Name="relativePath" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="rootDirectory">To be added.</param>
-        <param name="relativePath">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="rootDirectory">The Windows Runtime <see xref="T:Windows.Storage.IStorageFile"/> object that contains the file to read from.</param>
+        <param name="relativePath">The path, relative to the root folder, to the file to read from.</param>
+        <summary>Retrieves a stream for reading from a file in the specified parent folder.</summary>
+        <returns>A task that represents the asynchronous read operation.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+          #### Usage Note
+In Visual Basic and C#, you can call this method as an instance method on any object of type IStorageFolder. When you use instance method syntax to call this method, omit the first parameter. For more information, see [Extension Methods (Visual Basic)](/dotnet/visual-basic/programming-guide/language-features/procedures/extension-methods) or [Extension Methods (C# Programming Guide)](/dotnet/csharp/programming-guide/classes-and-structs/extension-methods).
```

Comment:
Let's make this a note inside remarks. Also, links to conceptual content under dotnet should use the ~/docs convention so we can validate the links at build time

```suggestion
> [!NOTE]
> In Visual Basic and C#, you can call this method as an instance method on any object of type IStorageFolder. When you use instance method syntax to call this method, omit the first parameter. For more information, see [Extension Methods (Visual Basic)](~/docs/visual-basic/programming-guide/language-features/procedures/extension-methods.md) or [Extension Methods (C# Programming Guide)](~/docs/csharp/programming-guide/classes-and-structs/extension-methods.md).
```